flask_admin/contrib/sqla/view.py

```diff
@@ -611,6 +614,27 @@ def scaffold_form(self):
 
         return form_class
 
+    def scaffold_list_form(self):
+        """
+            Create form for the list view editable columns.
+            
+            The form is created using the existing get_form(),
+            but each field is wrapped in a WTForms FieldList. 
+        """
+        converter = self.model_form_converter(self.session, self)
+        form_class = form.get_form(self.model, converter,
+                                   base_class=self.form_base_class,
+                                   only=self.column_editable_list)
+        
+        # iterate FormMeta to get unbound fields
+        field_dict = {}
```

Comment:
Was thinking if we can move this common code into some helper function that's called by all backends?